CONTRIBUTING.rst

```diff
@@ -9,8 +9,8 @@ Contributing
 ============
 
 Contributions are welcome, and they are greatly appreciated! Every
-little bit helps, and credit will always be given. Additional documentation 
-can be found in the Invenio `maintainers guide <https://inveniordm.docs.cern.ch/maintenance/newcomers/#tools>`_.
+little bit helps, and credit will always be given. Additional documentation
+can be found in the `InvenioRDM community contribution guide <https://inveniordm.docs.cern.ch/community/>`_.
 
 
 Types of Contributions
```

Comment:
It's not directly related to the links....but could you remove the outdated python versions in the "Pull Request" section? Or maybe we link to the documentation instead?

Reply:
I went further and edited the "Get Started" part in addition to the "Pull Request Guidelines" part. Take another look to see if it makes sense.